oidc-ui/README.md

```diff
@@ -51,6 +51,22 @@ The application runs on PORT=3000 by default.
   - REACT_APP_SBI_FINGER_BIO_SUBTYPES: sbi subtypes for finger
   - REACT_APP_SBI_ENV: Value that needs to be passed into sbi /capture request's evn parameter.
   - REACT_APP_ESIGNET_API_URL: This will be internally resolved to Esignet services URL (`/v1/esignet`).
+  - REACT_APP_LINKED_TRANSACTION_EXPIRE_IN_SEC: Transaction expire time.
+  - REACT_APP_QRCODE_DEEP_LINK_URI: Deep link uri for the QR code.
+  - REACT_APP_QRCODE_APP_DOWNLOAD_URI: URL for the Inji app download.
+  - REACT_APP_QRCODE_ENABLE: Boolean vlaue true or false to enable QR code.
+  - REACT_APP_CONSENT_SCREEN_EXPIRE_IN_SEC: Timer on the consent page which will expire in given secs.
+  - REACT_APP_SBI_PORT_RANGE: Port range for sbi.
+  - REACT_APP_RESEND_OTP_TIMEOUT_IN_SEC: Timer to resend the otp.
+  - REACT_APP_SEND_OTP_CHANNELS: Through which channel we get the otp.
+  - REACT_APP_CAPTCHA_ENABLE: Components where captcha needs to be shown.
+  - REACT_APP_AUTH_TXN_ID_LENGTH: transaction ID length.
+  - REACT_APP_OTP_LENGTH: Length of the otp.
+  - REACT_APP_PASSWORD_REGEX: Password pattern using regex.
+  - REACT_APP_WALLET_LOGO_URL: URL for the logo in the wallet which is configurable.
```

Comment:
URL for the logo in the wallet QRCode. This feature supports cross-origin-enabled image URLs only.